app/experimenter/settings_test.py

```diff
@@ -0,0 +1,3 @@
+from .settings import *
+
+UPLOADS_FILE_STORAGE = "inmemorystorage.InMemoryStorage"
```

Comment:
You know we had one of these and then got rid of it a while back.  You can also decorate the relevant classes with the override.  I'm hesitant to add this back in now just for one setting.

Reply:
I tried decorating individual tests. Two problems with that:

1. This is just about everything under nimbus models, since I added a factory to generate screenshots with branches. 

2. The storage is cached and gets used at the time of model class definition. I found that per-test or per-class override of storage type had no effect, so I ended up resorting to this.

Could poke at more of a lazy-loading storage class, but figured this was simpler.

Reply:
Oh yeah, fair point.  Okay yeah better to get something working landed and figure it out later if it becomes a hassle.  The reason we got rid of it was every so often you'd forget it exists and have to poke around to figure out where something is getting set, but yeah I see your point it happens at init time so too late to decorate.